packages/edit-widgets/src/blocks/legacy-widget/edit/handler.js

```diff
@@ -84,12 +84,11 @@ class LegacyWidgetEditHandler extends Component {
 			title = widgetTitle;
 		}
 
+		const RootComponent = isVisible ? 'div' : VisuallyHidden;
 		return (
-			<div
-				className={ classNames(
-					'wp-block-legacy-widget__edit-handler',
-					{ 'is-hidden': ! isVisible }
-				) }
+			<RootComponent
+				key="edit-handler"
```

Comment:
Do we need this `key`?

Reply:
We do, otherwise re-rendering would re-mount children and break TinyMCE.